layouts/shortcodes/changelog/changelogs.html

```diff
@@ -0,0 +1,10 @@
+{{/*
+   This template is used to render all of the changelog sections under
+   "content/changelogs"
+*/}}
+
+{{ with .Page.Resources.Match "*.md" }}
+{{ range ((sort . "Name" "desc")) }}
```

Comment:
will this cause a problem when we get to v1.10 or v10.0?

Reply:
bleh. yes.

Reply:
I've fixed this by including the timestamp of when the changelog was generated in the file, and sorting by that instead.